apps/web/src/design-system/table/Table.tsx

```diff
@@ -131,7 +131,7 @@ export function Table<T extends object>({
                 onClick={() => (!loading && onRowClick ? onRowClick(row) : null)}
                 {...row.getRowProps()}
                 className={classes.tableRow}
-                data-disabled={loading}
+                data-disabled={loading || !onRowClick}
```

Comment:
when the `onRowClick` is not defined the table is not clickable, so the cursor will be default